patchwork/steps/SimplifiedLLM/SimplifiedLLM.py

```diff
@@ -49,15 +56,31 @@ def __json_loads(json_str: str) -> dict:
             logger.debug(f"Json to decode: \n{json_str}\nError: \n{e}")
             raise e
 
+    @staticmethod
+    def w(text: str) -> str:
```

Comment:
this can probably be private by adding two underscores. Pick a better name as well.